samples/bluetooth/broadcast_config_tool/src/main.c

```diff
@@ -1844,6 +1894,69 @@ static int cmd_pd(const struct shell *shell, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_fixed_id(const struct shell *shell, size_t argc, char **argv)
+{
+	uint8_t big_index;
+
+	if (argc < 3) {
+		shell_error(
+			shell,
+			"Usage: bct fixed_id <0/1> <BIG index> [broadcast_id in hex (3 octets)]");
+		return -EINVAL;
+	}
+
+	if (!is_number(argv[1])) {
+		shell_error(shell, "Fixed ID must be a digit");
+		return -EINVAL;
+	}
+
+	uint8_t fixed_id = (uint8_t)atoi(argv[1]);
+
+	if (fixed_id != 0 && fixed_id != 1) {
+		shell_error(shell, "Fixed ID must be 0 or 1");
+		return -EINVAL;
+	}
+
+	if (!is_number(argv[2])) {
+		shell_error(shell, "BIG index must be a digit");
+		return -EINVAL;
+	}
+
+	big_index = (uint8_t)atoi(argv[2]);
+
+	if (big_index >= CONFIG_BT_ISO_MAX_BIG) {
+		shell_error(shell, "BIG index out of range");
+		return -EINVAL;
+	}
+
+	if (fixed_id) {
+		if (argc < 4) {
+			shell_error(shell, "Broadcast ID must be set");
+			return -EINVAL;
+		}
+
+		uint8_t len;
+
+		if (!is_hex(argv[3], &len)) {
+			shell_error(shell, "Broadcast ID must be a hex value");
```

Comment:
```suggestion
			shell_error(shell, "Broadcast ID must be a hexadecimal value");
```

Reply:
Not applied?

Reply:
Correct. Elected to shorten this for the error print only. I changed for the documentation though

Reply:
Fixed now, but didn't change for line:1904 as it is quite a long  print already